tensorflow/java/src/main/java/org/tensorflow/op/Inputs.java

```diff
@@ -0,0 +1,47 @@
+/* Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+package org.tensorflow.op;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.tensorflow.Input;
+import org.tensorflow.OperationBuilder;
+import org.tensorflow.Output;
+
+/** Utilities for manipulating input related types and lists. */
```

Comment:
Ok, got it ;) Well, my opinion on this is pretty much the same as yours: all this `Output/Input` conversion is confusing as both terms sound contradictory but are used to represent the exact same thing, one being a concrete class and the other an interface.

`Operand` is more "neutral" so might effectively result in less brain glitches, I like it. But I think it would be nice to exchange and think more about this, can we address this after as a separate issue?